main/integration-tests/__test__/api-tests/common/migration/my-study-migration.test.js

```diff
@@ -0,0 +1,95 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../../support/setup');
+const errorCode = require('../../../../support/utils/error-code');
+
+describe('Migration for My Studies scenarios', () => {
+  let setup;
+  let adminSession;
+  let defaultUser;
+  let defaultUserSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+    // Create cognito user to migrate ownership to
+    // Proper way that currently doesn't work:
+    // const username = setup.gen.username();
+    // defaultUser = adminSession.resources.users.defaults({ username });
+    // defaultUser = await await adminSession.resources.users.create(defaultUser);
+    // await adminSession.resources.users.user(defaultUser.uid);
+    // Workaround:
+    defaultUserSession = await setup.createResearcherSession();
+    defaultUser = defaultUserSession.user;
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Getting My Studies', () => {
+    it('should fail if request not by admin', async () => {
+      const researcherSession = await setup.createResearcherSession();
+      await expect(researcherSession.resources.migration.listInternaAuthUserMyStudies()).rejects.toMatchObject({
+        code: errorCode.http.code.forbidden,
+      });
+    });
+
+    it('should return message that there is nothing to migrate', async () => {
+      await expect(adminSession.resources.migration.listInternaAuthUserMyStudies()).resolves.toStrictEqual(
+        'No My Studies are owned my internal users. No migration needed.',
```

Comment:
I thought we discussed that this message was helpful to the admin users who had to migrate these resources. Should the API be returning an empty list with no helpful message like this?

Reply:
I don't think a message is required, the right place for this would rather be in this API's description field in the openapi file. Also we're explaining how to use this API in our release [documentation](https://github.com/awslabs/service-workbench-on-aws/pull/892/files#diff-3e9263816b528a0d25a0bf4c15d2d8721460ce24df019793b240a4c146f88186) as well.

The regular list studies API (not migrate list) can always be used to get a glimpse of their current state of study resources. Moreover, if the user has reached a point where they get back an empty list with this API - then they've already used this correctly, or they don't need to use it at all.

Reply:
The documentation mentions this message. I can edit that and include that change in this PR as well. Changing to empty list everywhere!